app/canard/rev2/fin_calib.h

```diff
@@ -0,0 +1,72 @@
+/*******************************************************************************
+*                                                                              *
+* FILE:                                                                        * 
+* 		    idle.h                                                             *
+*                                                                              *
+* DESCRIPTION:                                                                 * 
+* 		Idle state in FSM -- Logic incomplete                                  *
+*                                                                              *
+*******************************************************************************/
+
+/* Define to prevent recursive inclusion -------------------------------------*/
+#ifndef __MAIN_H
+#define __MAIN_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/*------------------------------------------------------------------------------
+ Includes                                                                    
+------------------------------------------------------------------------------*/
```

Comment:
I assume you copypasted main header file, which is good. But please delete unnecessary libraries for abort. Since abort is not yet to be developed so you can delete all libraries for now